tedge/src/cli/connect/bridge_config_c8y.rs

```diff
@@ -40,7 +40,7 @@ impl From<BridgeConfigC8yParams> for BridgeConfig {
             use_agent: true,
             try_private: false,
             start_type: "automatic".into(),
-            clean_session: true,
+            clean_session: false,
```

Comment:
I just noticed that comment from @rina23q : https://github.com/thin-edge/thin-edge.io/pull/417#discussion_r707396160

Reply:
Since this an info that this flag might not be correctly supported by Cumulocity, I do think:

* it's better to set `clean_session: false`, just being aware that there is no guarantees.